### ISSUE TITLE
Yuzu crashes everytime

### ISSUE THREAD

**Issue by Marionut84** [amakvana/YuzuModDownloader#20]
Yuzu crashes everytime I load it or do anything since installing yuzu mod downloader. Mods do not show up in Add-ons under preferences The 1st time I got it to pull up. Ever since it crashes the minute I do anything. Load a game, click on any setting or click on preferences. It immediately crashes. Have reinstalled everything.

**Comment by amakvana** [comment]
Just to clarify, is it YuzuModDownloader that crashes or is it Yuzu that crashes?

**Comment by Marionut84** [comment]
Just to clarify further, Yuzu crashes but it never did this before until I installed yuzu mod downloader. It worked perfect for months and since installing the downloader it crashes the second I click anything. I havent done anything new or different either. Just installed the mod downloader. If it helps at all the one and only time I got it to not crash at first, I had clicked on properties and there were no mods listed in add-ons, even though the downloader stated that mods for 23 games were successfully installed but there were no mods there. So I closed out restarted and ever since it crashes instantly everytime. Can't even click on a game. Very frustrating. Completely stumped.

**Comment by amakvana** [comment]
How strange?! Okay fair enough, it sounds like maybe one of the Yuzu config files have been corrupted. YuzuModDownloader doesn't touch any configurations files, however, let's see if we can sort it  

Can you post the following things:

1. Full path of where your yuzu.exe file resides?
2. Does the folder containing yuzu.exe, also contain a folder called "User"?

**Comment by Marionut84** [comment]
Sorry will send info shortly. Been right out straight the last couple of days with the kiddos. We'll have some free time here in just a bit. Thanks.

**Comment by Marionut84** [comment]
Hi sorry for the wait. So to answer your second question first no the it does not contain anything as far as a folder called user. There are only 2 folders inside of the exe folder and they are mediaservice and plugins

So the full path is:

C:\Users\owner\appdata\local\yuzu\yuzu-windows-msvc

Yuzu.exe is inside of that

 I've also tried multiple more things and anything that I could think of tonight including deleting my yuzu exe folder and reinstalling plus completely replacing my data folder where my mods and game info is stored with an original backup I have from before everything. I completely removed all of The Mods from the folders the yuzu Mod downloader had installed, one by one and then tried it again to no avail and I also reinstalled the mod downloader with a fresh reinstall of mods while selecting to delete any old mods and still yuzu exits upon doing anything with a game. Tonight I found though if I don't click on a game or anything like that then I can click on configure and get into the yuzu settings but I played around with everything and changed stuff and nothing is fixing it. The minute I click on a game or click on configure a game to check the ad-ons it exits out immediately. I'm really stumped and have no idea what to do. Like I said this was working great as of just a few days ago and now yuzu is completely unplayable. The only thing different that I have done is install and add the mod downloader Thinking it was a good thing to have and I know you said it does not effect system files but something is wrong and that's the only thing that's been done. I was literally just using it minutes before I added the downloader to the yuzu exe folder. I'm at a lost and don't know much more knowledge wise about this stuff To say that it's anything else specific. I'm just hoping for some help to figure it out Because it has to be something I have no clue about. Either way thanks for your responses back. I appreciate it.

**Comment by amakvana** [comment]
That's very strange, to me it sounds like there could be a corruption within one of the settings files. 

Try this for me:
1. Close Yuzu
2. Start > Command Prompt
3. Copy and Paste the following commands below into Command Prompt, pressing Enter after each one:
`ren "%appdata%\yuzu" "yuzu.old"`
`md "%appdata%\yuzu\keys"`
`copy /y "%appdata%\yuzu.old\keys\prod.keys" "%appdata%\yuzu\keys\prod.keys"`
4. Open Yuzu again 

You may have to configure your game path again, along with changing the API to Vulkan, do this and check you can run games again

The steps above just clears any old settings that may have been corrupted

**Comment by Marionut84** [comment]
Hi thanks for getting back to me. So after doing some more research on the Internet last night and after trying everything I finally figured it out. It was so simple and I think upon reading your comment today you actually came up with the same solution just a little more complicated way of doing it but it sounds like it would do the trick. Anyways all I had to do was delete the global configuration file. I had to reselect everything and tweak it a bit but everything is working now and really good I might add. All the mods are there, downloader is working properly. I appreciate You trying to help me figure out. Thank you again.

**Comment by amakvana** [comment]
Excellent! Glad that's fixed your problem

**Comment by Marionut84** [comment]
Hi. Sorry for the wait in response. I thought I had posted this earlier but just realized I hadn't. I wanna say thanks for getting back to me too.  I appreciate you trying to help me troubleshoot. So after doing some more research on the Internet last night and after trying everything I finally figured it out. It was so simple and I think upon reading your comment today you actually came up with the same solution just a little more complicated way of doing it but it sounds like it would do the trick as well. Anyways all I had to do was delete the global configuration file. I had to reselect everything and tweak it a bit but everything is working now and really good I might add. All the mods are there, downloader is working properly. I appreciate You trying to help me figure out. It was literally as simple as deleting an INI file. So Thank you again.
